docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,87 @@
+
+### Design
+
+To solve this problem, we will implement a procedure called Proportional Slashing.  The desire is that the larger a validator is, the more they should be slashed.  The first naive attempt is to make a validator's slash percent proportional to their share of consensus voting power.
+
+```
+slash_amount = power // power is the faulting validator's voting power.
+```
+
+However, this will incentivize validators with large amounts of stake to split up their voting power amongst accounts, so that if they fault, they all get slashed at a lower percent.  The solution to this is to take into account not just a validator's own voting percentage, but also the voting percentage of all the other validators who get slashed in a short time window.
+
+```
+slash_amount = (power_1 + power_2 + ... + power_n) // where power_i is the voting power of the ith validator faulting in the period
+```
+
+Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same window, they both will still get slashed at the sum 10% amount.
+
+However, an operator might still choose to split up their stake across multiple accounts with hopes that if any of them fault independently, they will not get slashed at the full amount.  In the case that the validators do fault together, they will get slashed the same amount as if they were one entity.  There is no con to splitting up.  However, if operators are going to split up their stake without actually decorrelating their setups, this also causes a negative externality to the network as it fills up validator slots that could have gone to others or increases the commit size.  In order to disincentivize this, we want it to be the case such that splitting up a validator into multiple validators and they fault together is punished more heavily that keeping it as a single validator that faults.
+
+We can achieve this by not only taking into account the sum of the percentages of the validators that faulted, but also the *number* of validators that faulted in the window.  One general form for an equation that fits this desired property looks like this:
+
+```
+slash_amount = (sqrt(power_1) + sqrt(power_2) + ... + sqrt(power_n))^2
+```
+
+So now, for example, if one validator of 10% faults, it gets a 10% slash, while if two validators of 5% each fault together, they both get a 20% slash ((sqrt(0.05)+sqrt(0.05))^2).
+
+One will note, that this model doesn't differentiate between multiple validators run by the same operators vs validators run by different operators.  This can be seen as an additional benefit in fact.  It incentivizes validators to differentiate their setups from other validators, to avoid having correlated faults with them or else they risk a higher slash.  So for example, operators should avoid using the same popular cloud hosting platforms or using the same Staking as a Service providers.  This will lead to a more resilient and decentralized network.
```

Comment:
```
This can be seen as an additional benefit in fact.  It incentivizes validators to differentiate their setups from other validators, to avoid having correlated faults with them or else they risk a higher slash.
```

idk if this is fair to say exactly. There are only so many cloud providers and so many correlations you can make. It can still be the case that a lowly validator gets bucketed with a top 10 validator with a completely different setup.

Reply:
Suppose, the validator having power 100. he got delegations from two accounts with `min-self-delegation` = 1atom (he got remaining 99atoms from delegators). In this case how the Proportional slashing will work?

Reply:
@baymax19 proportional slashing makes no distinction between self-bond vs external delegation.  A validator's total delegation is what is taken into account.

Reply:
@alexanderbez  We should encourage validators to not be overly reliant on cloud providers, especially on single cloud providers, as that can be a large source of centralization risk to the network.  Furthermore, even on cloud providers, you can do things like diversify with respect to regions, OSs, access control, etc.

Most importantly, however, it should hopefully decrease reliance on "Staking as Service Operators", entities that run white-label validation services for other validators.